sample_xblocks/basic/problem.py

```diff
@@ -117,13 +117,29 @@ def student_view(self, context=None):
                     }
                 }
 
-                function handleCheckResults(results) {
-                    $.each(results.submitResults || {}, function(input, result) {
-                        callIfExists(runtime.childMap(element, input), 'handleSubmit', result);
-                    });
-                    $.each(results.checkResults || {}, function(checker, result) {
-                        callIfExists(runtime.childMap(element, checker), 'handleCheck', result);
-                    });
+                function handleCheckResults(data) {
+                    return function(results) {
+                        $.each(results.submitResults || {}, function(input, result) {
+                            callIfExists(runtime.childMap(element, input), 'handleSubmit', result);
+                        });
+                        answers = {};
```

Comment:
Thank you. Fixed (will push in a moment). I broke the .js file out so I could run jshint for other places I had similar errors.